docs/source/quick_start.rst

```diff
@@ -0,0 +1,175 @@
+Quick Start Guide
+==================
+
+If you haven't already done so, see installation instructions in :doc:`Getting Started <./getting_started>`. For this guide, you will need to checkout Maestro via GitHub in order to use the provided samples.
+
+Running the LULESH Study
+*************************
+
+This section will take you through the basics of using Maestro to launch a study locally. If you're looking for a more detailed breakdown of the LULESH study, skip to the :doc:`LULESH Specification Breakdown <./lulesh_breakdown>`.
+
+`LULESH <https://github.com/LLNL/LULESH>`_ (Livermore Unstructured Lagrangian Explicit Shock Hydrodynamics) is a proxy application developed and open sourced by Lawrence Livermore National Laboratory. The application is capable of being compiled for both serial and distributed execution and provides a simple stand-in for an actual simulation code.
+
+The LULESH study comes in two flavors: one for Unix systems and one for MacOSX systems. Both specifications are just about identical save for minor differences in ``sed`` commands. Simply pick the version for your system. In order to execute the LULESH sample study, execute the following command using Maestro from the root of the repository (using the Unix version as an example)::
+
+    $ maestro run ./samples/lulesh/lulesh_sample1_unix.yaml -o ./tests/lulesh

```

Comment:
Consider adding a note describing the default behavior if `-o` is not specified.